frontend/src/views/Plaza.vue

```diff
@@ -2,6 +2,19 @@
   <div class="body main-font">
 
     <div v-if="ownCharacters.length === 0" class="blank-slate">
+      <div class="current-promotion">
+        <div class="tob-bg-img promotion-decoration">
+          <img class="vertical-decoration bottom" src="../assets/border-element.png">
+        </div>
+        <strong>SIGN UP BONUS</strong>
```

Comment:
please don't do uppercase text in code - add a class that does text-transform: uppercase